### PR TITLE
Initial cleanup of backend integration tests

### PR DESCRIPTION
**Pull request by calebmshafer**
I split out a handful of changes I have to the tests into multiple PRs, this will be the first pass to clean up some of the setup and reduce duplication across the test suite.

All the tests are still not passing (on this branch or `master`), I'm getting together a list and tracking down each one.

This PR also depends on #914

### CODE REVIEW

File: core/backend/src/test/integration/Checkpoints.test.ts [hunk 1]
```diff
@@ -16,12 +16,9 @@ import { HubUtility } from "./HubUtility";
 
 // FIXME: Disabled because V2 checkpoints are not in QA yet...
```
Comment (pmconne) [inline core/backend/src/test/integration/Checkpoints.test.ts:17]: Tangential to this PR but I think this comment is no longer accurate.

Comment (calebmshafer) [reply core/backend/src/test/integration/Checkpoints.test.ts:17]: I'll verify with @wgoehrig but I think that's still accurate.  v2 checkpoints are in DEV currently, hopefully QA by EOW.

**Review by pmconne — APPROVED** [review]

File: core/backend/src/test/integration/Checkpoints.test.ts (hunk 1 above)
Comment (wgoehrig) [reply core/backend/src/test/integration/Checkpoints.test.ts:17]: yeah, @calebmshafer, that's right.

**Review by wgoehrig — APPROVED** [review]

File: core/backend/src/test/integration/ChangedElements.test.ts [hunk 2]
```diff
@@ -2,63 +2,57 @@
...
+
+    testContextId = await HubUtility.getTestContextId(requestContext);
+    requestContext.enter();
+    testIModelId = await HubUtility.getTestIModelId(requestContext, HubUtility.testIModelNames.readOnly);
+    requestContext.enter();
 
     // Purge briefcases that are close to reaching the acquire limit
-    const managerRequestContext = await TestUtility.getAuthorizedClientRequestContext(TestUsers.manager);
-    await HubUtility.purgeAcquiredBriefcases(managerRequestContext, "iModelJsIntegrationTest", "ReadOnlyTest");
+    await HubUtility.purgeAcquiredBriefcasesById(requestContext, testIModelId);
```
Comment (ramanujam-raman) [inline core/backend/src/test/integration/ChangedElements.test.ts:28]: The regular user was supposed to have limited permissions. Not sure if that changed now - but that was the reason to use manager here.

**Review by ramanujam-raman — APPROVED** [review]

File: core/backend/src/test/integration/ChangedElements.test.ts (hunk 2 above)
Comment (calebmshafer) [reply core/backend/src/test/integration/ChangedElements.test.ts:28]: Okay, I thought an individual user had access to free their own briefcases and didn't need someone else to do it for them.  I'll test that out to make sure it works.

**Pull request merged**